pkgs/development/python-modules/vdirsyncer/default.nix

```diff
@@ -24,21 +24,21 @@
 
 buildPythonPackage rec {
   pname = "vdirsyncer";
-  version = "0.19.0";
+  version = "0.19.1";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256:0995bavlv8s9j0127ncq3yzy5p72lam9qgpswyjfanc6l01q87lf";
+    hash = "sha256-qnbHclqlpxH2N0vFzYO+eKrmjHSCljWp7Qc81MCfA64=";
   };
 
   postPatch = ''
     substituteInPlace setup.py \
       --replace "click-log>=0.3.0, <0.4.0" "click-log>=0.3.0, <0.5.0"
 
-    sed -i -e '/--cov/d' -e '/--no-cov/d' setup.cfg
+    sed -i -e '/--cov/d' -e '/--no-cov/d' pyproject.toml
```

Comment:
So `format = "pyproject"`?

Reply:
seems like both "pyproject" and "setuptools" work but I changed to "pyproject" because the author has made 2 commits to migrate stuff to pyproject.toml.